os/dutils.sh

```diff
@@ -57,6 +60,64 @@ function FIND_DBGBINFILE()
 	KERNEL_DBG_BIN="tinyara$EXEEXT"
 }
 
+function TRAP_MENU()
+{
+	echo ======================================================
+	echo "  \"Run TRAP tool for\""
+	echo ======================================================
+	echo "  \"1. Default binary\""
+	echo "  \"2. Custom binary\""
```

Comment:
I think it's difficult to understand. If we can't get understandable short sentence, let's write full sentence.